src/main/webapp/WEB-INF/tags/student/studentPage.tag

```diff
@@ -15,7 +15,7 @@
   <t:bodyFooter />
 </c:set>
 
-<t:page pageTitle="${pageTitle}" bodyTitle="${bodyTitle}">
+<t:page pageTitle="${pageTitle} [Student]" bodyTitle="${bodyTitle}">
```

Comment:
Perhaps student pages don't need a suffix at all, as that is probably the most common usage (in terms of traffic) of TEAMMATES. Would @damithc be agreeable?